cve_bin_tool/checkers/libevent.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+
+CVE checker for Libevent
+
+https://www.cvedetails.com/vulnerability-list/vendor_id-15590/product_id-32303/Libevent-Project-Libevent.html
+
+"""
+
+from cve_bin_tool.checkers import Checker
+
+
+class LibeventChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"Installation directory for CMake files",
+        r"if Libevent should not be compiled with thread support",
+        r"if Libevent should build without support for OpenSSL encryption",
+        r"if Libevent should build without support for a debug mode",
+    ]
+
+    FILENAME_PATTERNS = [r"libevent"]
+    VERSION_PATTERNS = [
+        r"libevent-([01]\.[0-3]\)",
+        r"libevent-([1-9]+\.[0-9]+\.[0-9]+-stable)",
+        r"libevent-([1-9]+\.[0-9]+\.[0-9]+-alpha)",
+        r"libevent-([1-9]+\.[0-9]+\.[0-9]+-beta)",
+        r"libevent-([1-9]+\.[0-9]+\.[0-9]+-rc)",
```

Comment:
```suggestion
        r"libevent[- ]([0-9]+\.[0-9]+(\.[0-9]+)?)",
```

This pattern covers all the 5 patterns and also the test case string with space instead of `-`.

Reply:
Thank you @BreadGenie, updated the pr.